packages/thirdweb/src/exports/react.ts

```diff
@@ -1,3 +1,5 @@
+"use client";
```

Comment:
do we *need* it here because not everything that's exported from here needs to be client only?

Reply:
you're right, meant to remove that one